kuwala/core/neo4j/importer/src/main.py

```diff
@@ -7,6 +7,7 @@
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
+    parser.add_argument('--date',help='Selected update date to process')
```

Comment:
The argument should also be more contextualized since they are provided for the importer in general and not just the population density pipeline. So `--population-density-date` would be a more suitable name.

Reply:
ATTENTION: This leads to downstream changes in the population-density importer.